tests/conftest.py

```diff
@@ -45,8 +45,14 @@ def generate_key_pair():
 
     @staticmethod
     def read_private_key():
+        """
+        Add your own rsa private key(for testing only) in tests directory
+        Follow instructions here
+        https://docs.snowflake.net/manuals/user-guide/data-load-snowpipe-rest-gs.html#using-key-pair-authentication
+        :return: private key pem encoded
+        """
         tests_dir = os.path.dirname(os.path.realpath(__file__))
-        private_key_filename = os.path.join(tests_dir, "sfctest0_private_key_1.p8")
```

Comment:
Please don't change this. Travis tests use this private key file

Reply:
My bad, I changed this locally, shouldnt be a part of this PR.